CesiumAsync/src/CachingAssetAccessor.cpp

```diff
@@ -343,8 +347,14 @@ std::string calculateCacheKey(const IAssetRequest& request) {
 std::time_t calculateExpiryTime(
     const IAssetRequest& request,
     const std::optional<ResponseCacheControl>& cacheControl) {
+
+  // Prefer cache control directives over older Expires header
   if (cacheControl) {
-    if (cacheControl->maxAge() != 0) {
+    // A specified maxAge indicates exact expiration.
+    // A specified staleWhileRevalidate still indicates expiration with maxAge,
+    // even if it is 0.
```

Comment:
`stale-while-revalidate` doesn't actually indicate anything about expiration. It just declares a (kinda complicated) policy for what you can do with it _after_ it's expired. We can ignore that special policy for now, which really means ignoring `stale-while-revalidate` and instead revalidating immediately and always.

Reply:
Agreed. I've changed this.